azext_iot/central/commands_device.py

```diff
@@ -679,3 +684,191 @@ def get_downstream_rel_name(
                 rel_name = content.get("name")
 
     return rel_name
+
+
+def get_attestation(
+    cmd,
+    app_id: str,
+    device_id,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.ga_2022_05_31.value,
+) -> dict:
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+    return provider.get_device_attestation(
+        device_id=device_id,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+
+def delete_attestation(
+    cmd,
+    app_id: str,
+    device_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.ga_2022_05_31.value,
+) -> dict:
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.delete_device_attestation(device_id, central_dns_suffix=central_dns_suffix)
+
+
+def update_attestation(
+    cmd,
+    app_id: str,
+    device_id: str,
+    content: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.ga_2022_05_31.value,
+) -> dict:
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    payload = utility.process_json_arg(content, argument_name="content")
+
+    return provider.update_device_attestation(device_id, payload=payload, central_dns_suffix=central_dns_suffix)
+
+
+def create_attestation(
+    cmd,
+    app_id: str,
+    device_id: str,
+    content: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.ga_2022_05_31.value,
+) -> dict:
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    payload = utility.process_json_arg(content, argument_name="content")
+
+    return provider.create_device_attestation(device_id, payload=payload, central_dns_suffix=central_dns_suffix)
+
+
+def list_modules(
+    cmd,
+    app_id: str,
+    device_id,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.ga_2022_05_31.value,
+) -> dict:
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.list_modules(
+        device_id=device_id,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+
+def list_components(
+    cmd,
+    app_id: str,
+    device_id,
+    module_name=None,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.ga_2022_05_31.value,
+) -> dict:
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.list_device_module_components(
```

Comment:
this provide method is confuse here with list_device_module_components, I suppose it will be list_device_components, if module is not null then return from module

Reply:
Sure. I'll update it.